R/init-package.R

```diff
@@ -11,14 +11,20 @@
 
   .addPackageLibToPath()
 
-  rClr::clrLoadAssembly(filePathFor("OSPSuite.R.dll"))
+  if (.Platform$OS.type == "unix") {
+    dyn.load(system.file("lib","libOSPSuite.SimModelSolver_CVODES.so", package = ospsuiteEnv$packageName))
+    dyn.load(system.file("lib","libOSPSuite.FuncParserNative.so", package = ospsuiteEnv$packageName))
+    dyn.load(system.file("lib","libOSPSuite.SimModelNative.so", package = ospsuiteEnv$packageName))
+  }
```

Comment:
with this, adding **lib** folder to LD_LIBRARY_PATH under Linux will not be required anymore?

Reply:
Correct

Reply:
Cool 👍